src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -0,0 +1,182 @@
+                  }}
+                />
+                <GeneTranscriptInfoInput
+                  referenceGenome={REFERENCE_GENOME.GRCH38}
+                  isoform={props.data?.isoform_override_grch38}
+                  refseq={props.data?.dmp_refseq_id_grch38}
+                  onIsoformChange={e => {
+                    props.updateReviewableContent(firebaseGenePath, 'isoform_override_grch38', e.target.value);
+                  }}
+                  onRefseqChange={e => {
+                    props.updateReviewableContent(firebaseGenePath, 'dmp_refseq_id_grch38', e.target.value);
+                  }}
+                />
+              </div>
+              <RealtimeBasicInput
+                label="Summary"
+                labelClass="font-weight-bold"
+                name="geneSummary"
+                type={RealtimeInputType.TEXTAREA}
+                value={props.data?.summary || ''}
+                onChange={e => {
+                  props.updateReviewableContent(firebaseGenePath, 'summary', e.target.value);
+                }}
+              />
+              <div className="flex d-flex">
+                <RealtimeBasicInput
+                  label="Tumor Supressor"
+                  name="tumorSupressor"
+                  type={RealtimeInputType.CHECKBOX}
+                  checked={!!props.data?.type?.tsg}
+                  onChange={e => {
+                    props.updateGeneType(firebaseGenePath, TUMOR_SUPPRESSOR, e.target.checked);
+                  }}
+                />
+                <RealtimeBasicInput
+                  className="ml-4"
+                  label="Oncogene"
+                  name="oncogene"
+                  type={RealtimeInputType.CHECKBOX}
+                  checked={!!props.data?.type?.ocg}
+                  onChange={e => {
+                    props.updateGeneType(firebaseGenePath, ONCOGENE, e.target.checked);
+                  }}
+                />
+              </div>
+            </Col>
+          </Row>
+          <Row className={getSectionClassName()}>
+            <Col>
+              <RealtimeBasicInput
+                label="Background"
+                labelClass="font-weight-bold"
+                name="geneBackground"
+                type={RealtimeInputType.TEXTAREA}
+                value={props.data?.background || ''}
+                onChange={e => {
+                  props.updateReviewableContent(firebaseGenePath, 'background', e.target.value);
+                }}
+              />
+              <div className="mb-2">
+                <AutoParseRefField summary={props.data?.background} />
+              </div>
+              <div>
+                <span className="font-weight-bold mr-2">External Links:</span>
+                <WithSeparator separator={InlineDivider}>
+                  <a href={`https://cbioportal.mskcc.org/ln?q=${props.data?.name}`} target="_blank" rel="noopener noreferrer">
+                    CBioPortal <ExternalLinkIcon />
```

Comment:
```suggestion
                    cBioPortal <ExternalLinkIcon />
```

Reply:
Added `cBioPortal` and `COSMIC` strings to the `constants.ts` file.